prow/getting_started_deploy.md

```diff
@@ -15,6 +15,8 @@ You need three things:
 2) a [personal access token][1] for the github bot user you want to act as prow
 
    - Prow uploads this token into a secret in your cluster
+   - The token should be saved in plain text
+   - The token must grant prow write/push access the GitHub repos. More details on GitHub ACLs are available [here][8]. 
```

Comment:
We should keep this brief and not conflate user access from token scopes:

- Bot user must have write access to the repo.
- Token must have `public_repo` and `repo:status scopes` (and `repo` for private repos), more details "here"

These two instructions aren't entirely correct for all plugins (some plugins list org members, which will require the user to be able to do this) but is probably good enough for getting started

We should probably update both this line and https://github.com/kubernetes/test-infra/blob/9c2d9623c2a1e55a3e8eeecce31531670ef29381/prow/getting_started_deploy.md#bot-account to point to the same here location with more detailed info.

Reply:
ok, thanks.